docs/index.rst

```diff
@@ -6,6 +6,13 @@
 D-Wave Binary CSP
 =================
 
+.. deprecated:: 0.3.1
+
+    ``dwavebinarycsp`` is deprecated and will be removed in Ocean 10.
+    For solving problems with constraints, we recommand using the hybrid
+    solvers in the Leap service.
```

Comment:
Can we recommend (or link to) a non-HSS solution? For someone wanting to say in the QPU-land. Or that wouldn't make sense?

Reply:
IMO for constrained problems we want to steer them to the HSS. There are some alternatives when using the QPU directly but those are rather specific, and I don't think those users would be coming from here.